configure

```diff
@@ -3789,6 +3794,26 @@ case $_host in
 		;;
 esac
 
+#
+# Enable Lua only for engines that support it
```

Comment:
“Enable LUA only for engines that require it”

Reply:
"Lua" is actually correct. It's not an acronym.

Reply:
You are quite correct, my mistake. It's "Lua"